### PR TITLE
Update adding-a-new-ssh-key-to-your-github-account.md

### PR DESCRIPTION
**Pull request by dhirschfeld**
Add example of adding a signing key

### Why:

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Support for adding signing keys was added in https://github.com/cli/cli/pull/7270

**Comment by nguyenalex836** [comment]
@dhirschfeld Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by aunzaa — COMMENTED** [review]
Good

**Comment by Lildevil35** [comment]
``authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md

**Review by Laceduppanda — COMMENTED** [review]
Yes let's go #31103

File: content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md [hunk 1]
```diff
@@ -63,12 +63,10 @@ After adding a new SSH authentication key to your account on {% ifversion ghae %
...
 Before you can use the {% data variables.product.prodname_cli %} to add an SSH key to your account, you must authenticate to the {% data variables.product.prodname_cli %}. For more information, see [`gh auth login`](https://cli.github.com/manual/gh_auth_login) in the {% data variables.product.prodname_cli %} documentation.
 
-{% ifversion ssh-commit-verification %}At present, you can only use {% data variables.product.prodname_cli %} to add SSH authentication keys, you cannot add SSH signing keys.{% endif %}
-
-To add an SSH authentication key to your GitHub account, use the `ssh-key add` subcommand, specifying your public key. If you're prompted to request additional scopes, follow the instructions in the command line.
+To add an SSH key to your GitHub account, use the `ssh-key add` subcommand, specifying your public key. For authentication keys, if you're prompted to request additional scopes, follow the instructions in the command line.
 
 ```shell
-gh ssh-key add KEY-FILE
+gh ssh-key add KEY-FILE --type {authentication|signing}
```
Comment (Laceduppanda) [inline content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md:69]: 0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826

**Review by hubwriter — APPROVED** [review]
@dhirschfeld - Looks good. 
Many thanks for updating the docs on this. 👍

**Pull request merged**